src/datasets/loaders/10x_datasets/config.vsh.yaml

```diff
@@ -0,0 +1,96 @@
+functionality:
+  name: 10x_datasets
```

Comment:
The CI fails because:

    requirement failed: functionality name must begin with a letter and consist only of alphanumeric characters or underscores.


Since it uses `squidpy.read.visium`, how about we rename this to `name: tenx_visium`?